tests/app/forms/test_questionnaire_form.py

```diff
@@ -16,1677 +16,1678 @@
 from app.questionnaire import QuestionnaireSchema
 from app.questionnaire.placeholder_renderer import PlaceholderRenderer
 from app.utilities.schema import load_schema_from_name
-from tests.app.app_context_test_case import AppContextTestCase
 
 
-# pylint: disable=no-member
-# wtforms Form parents are not discoverable in the 2.3.3 implementation
-class TestQuestionnaireForm(
-    AppContextTestCase
-):  # noqa: C901  pylint: disable=too-many-public-methods
-    @staticmethod
-    def _error_exists(answer_id, msg, mapped_errors):
-        error_id = f"{answer_id}-error"
-        return any(
-            e_id == error_id and str(msg) in ordered_errors
-            for e_id, ordered_errors in mapped_errors
-        )
+def _error_exists(answer_id, msg, mapped_errors):
```

Comment:
Wasn't the underscore prefix meant as a hint that this method is intended for internal use? Since its class doesn't exist anymore we can rename it.

Reply:
I suppose it's still a private-ish function, it's still the same concept as having it in a class. It's harmless either way - it's a test module and unlikely to be imported ever

Reply:
I was thinking we should do this here since you renamed `_test_validation` in `test_duration_form.py` but it's not crucially important.